src/styles/blocks/card.scss

```diff
@@ -0,0 +1,88 @@
+.card {
+  &__product-image {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__product-name {
+    margin-bottom: 4px;
+
+    font-size: 12px;
+    font-weight: 500;
+    font-style: normal;
+    line-height: 18px;
+
+    color: $font-color-dark;
+  }
+
+  &__product-code {
+    margin-bottom: 16px;
+
+    font-style: normal;
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: $font-color-light;
+  }
+
+  &__product-rating {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+
+    font-weight: 400;
+    font-size: 10px;
+  }
+
+  &__product-pricing {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+  }
+
+  &__product-price {
+    display: block;
+
+    font-weight: 400;
+    font-size: 12px;
+    line-height: 18px;
+    color: $font-color-light;
+  }
+
+  &__product-money-value {
+    display: block;
+
+    font-style: normal;
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+
+    color: $font-color-dark;
+  }
+
+  &__product-link {
+    box-sizing: border-box;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 40px;
+
+    font-weight: 700;
+    font-size: 14px;
+    text-transform: uppercase;
+    text-decoration: none;
+    color: #fff;
+
+    background: $blue-color;
+    border: 1px solid $blue-color;
```

Comment:
In previous task 'card' we should do transparent border and change its color on hover.